### PR TITLE
Added notes for 'What's new in Machine Learning' and 'Understanding Image in Vision'

### PR DESCRIPTION
**Pull request by gerriet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -911,3 +913,60 @@ https://developer.apple.com/wwdc19/615
   - teamPlayerID scoped to development team
   - gamePlayerID scoped to game
 - `loadPlayersForIdentifiers:withCompletionHandler:`
+
+## What's new in Machine Learning
+
+https://developer.apple.com/wwdc19/209
+
+- Gaurav Kapoor
```
Comment (Blackjacx) [inline README.md]: Who is that? Is this name intended? Maybe more description needed here.

**Review by Blackjacx — CHANGES_REQUESTED** [review]

**Comment by Blackjacx** [comment]
Super cool additions! Thanks so much :-) Could you please resolve the conflicts? Then it is a pleasure for me to merge it 👍

File: README.md (hunk 1 above)
Comment (gerriet) [reply README.md]: He held the talk. I added names for the presenters. In the other talk different segments by different presenters (so the names are in parenthesis after the segment topic), this one is only done by one person. Will add an "presented by".

Comment (Blackjacx) [reply README.md]: Ahh okay :-)

**Comment by gerriet** [comment]
OK, I fixed the conflicts. Thanks for the appreciation, I think you started a great thing here, thanks a lot!

**Review by Blackjacx — APPROVED** [review]
Perfect - thanks for your contribution :-)

**Comment by Blackjacx** [comment]
Thanks I'm happy that this project got so popular :)

**Pull request merged**